src/XMLexport.cpp

```diff
@@ -387,6 +387,7 @@ void XMLexport::writeHost(Host* pHost, pugi::xml_node mudletPackage)
     host.append_attribute("mThemePreviewItemID") = QString::number(pHost->mThemePreviewItemID).toUtf8().constData();
     host.append_attribute("mThemePreviewType") = pHost->mThemePreviewType.toUtf8().constData();
     host.append_attribute("mSearchEngineName") = pHost->mSearchEngineName.toUtf8().constData();
+    host.append_attribute("MinimumTimeForTimerDebugOutput") = pHost->mTimerDebugOutputSuppressionInterval.toString(QLatin1String("HH:mm:ss.zzz")).toUtf8().constData();
```

Comment:
I'd like to see the key be `mTimerSupressionInterval` for consistency with other keys.

Reply:
I'd like to see all the keys be named sensibly to reflect their purpose :neckbeard: - and this one does not "*suppress Timers for an interval*" - but making such a change for the existing attributes is not a trivial change - identifiers in XML should, IMHO, actually reflect what they do - but in the past some Mudlet coders have slavishly copied even BADLY_NAMED_SHOUTY_VARIABLES directly into the *attributes* for the `<Host ...>` and other tags - causing the mess we have nowadays.

The only possible reason for using the **same** name for the attribute in the XML file as the variable that it is tied to is: to keep it *simple* for the coder - it does not help an end user ~hacking~ fiddling with the file manually (or a separate XML parser trying to make sense of the file independently of Mudlet) - in those circumstances it is better *I think* to have names that reflect what they mean.

{*The difficulty with previously badly named identifiers is that it is not safe to change the identifier later unless it is duplicated during a transition period with the old one only being dropped when it is deemed appropriate to do so - usually after a *major* version number increment - which is a very similar point to one I was arguing about in a different PR about phasing out an awkward way of storing a particular Mudlet application setting and converting it to a more consistent and easier to work with method already used for other settings...!*}

Reply:
I'd rather just see consistency with the rest of the keys. This one clearly stands out. Someone else will come along to Mudlet and want some other style, and so on and so forth - consistency is better.

Reply:
Okay...